src/main/java/org/kcctl/command/RestartConnectorCommand.java

```diff
@@ -22,25 +22,51 @@
 import org.kcctl.service.KafkaConnectApi;
 import org.kcctl.util.ConfigurationContext;
 
+import picocli.CommandLine;
 import picocli.CommandLine.Command;
 import picocli.CommandLine.Parameters;
 
 @Command(name = "connector", description = "Restarts the specified connector")
 public class RestartConnectorCommand implements Runnable {
 
-    @Inject
-    ConfigurationContext context;
-
     @Parameters(paramLabel = "NAME", description = "Name of the connector (e.g. 'my-connector')", completionCandidates = ConnectorNameCompletions.class)
     String name;
 
+    @CommandLine.Option(names = { "-t", "--tasks" }, description = "Restart tasks either all or just failed")
+    String withTasks;
```

Comment:
Let's make this an enum, as e.g. [here](https://github.com/kcctl/kcctl/blob/main/src/main/java/org/kcctl/command/PatchLogLevelCommand.java#L43), that way options will be validated and we get tab completion too.